src/features/accept-bid/AcceptBidForm/FormSteps/ui/NFTDetails/NFTDetails.tsx

```diff
@@ -31,108 +33,41 @@ export const NFTDetails: FC<NFTDetailsProps> = ({ zna }) => {
 	const truncatedZna = truncateDomain(zna, 20);
 	const truncatedCreatorAddress = truncateAddress(creator);
 	const highestBidString = highestBid ? formatEthers(highestBid?.amount) : '-';
+	const mediaVariant = isMediaAnimated ? 'video' : 'image';
```

Comment:
You'll see this variant being used in the flows. This is to determine if the media asset is either a video or image. The reason i've added this is to adjust the height of the media. If an image, the media container will be larger as seen in some of the screen shots in this PR description, but if a video, the height is not set. This is to ensure the video controls sit nicely at the bottom of the video component and the media container is placed nicely in the modal. I think the reason for the differences is due to aspect ratios of the media assets

Reply:
Nice, makes sense. I wonder if there's a way to handle this more dynamically, such that we don't have to do this check when we render an asset of unknown type.